qa/tasks/cephfs/test_openfiletable.py

```diff
@@ -0,0 +1,41 @@
+import time
+from cephfs_test_case import CephFSTestCase
+from teuthology.exceptions import CommandFailedError
+from tasks.cephfs.cephfs_test_case import CephFSTestCase, for_teuthology
+
+class OpenFileTable(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def test_max_items_per_obj(self):
+        """
+        The maximum number of openfiles omap objects keys are now equal to
+        osd_deep_scrub_large_omap_object_key_threshold option.
+        """
+        self.set_conf("mds", "osd_deep_scrub_large_omap_object_key_threshold", "5")
+
+        self.fs.mds_restart()
+        time.sleep(10)
```

Comment:
Ah, yes. I think you need to restart after setting the config. Instead of `time.sleep`, use `self.fs.wait_for_daemons()`

Reply:
Done. Thanks, Patrick.